.changeset/serious-baboons-mate.md

```diff
@@ -0,0 +1,5 @@
+---
+'@guardian/braze-components': minor
```

Comment:
Maybe even `patch` - I don't think this will be visible to consumers at all? (Although I do like this changeset name 😂 )

Reply:
I'm going to push both PRs through together so it should amount to just one `minor` bump I think